### PR TITLE
Add pitch_string_analysis

### PR DESCRIPTION
**Pull request by ookamitai**
Added pitch_string_analysis :DDD

### CODE REVIEW

**Review by SuibianP — COMMENTED** [review]
Thanks a lot for the efforts!

The visualisation might be better generated as a vector image, which scales and versions more easily.

I do not currently have an available MkDocs installation at hand to verify (not willing to breach company policies). Could you please kindly enable GitHub Pages from your repository and trigger the GitHub Action workflow so that the effects can be seen from https://ookamitai.github.io/utau-zh-docs/?

As a side comment, did you mean to commit to the `add_pitch_decode` branch in your fork instead?

**Review by SuibianP — CHANGES_REQUESTED** [review]
Please append to [CONTRIBUTORS.md](https://github.com/ookamitai/utau-zh-docs/blob/doc-source/docs/CONTRIBUTORS.md) as well.

**Comment by ookamitai** [comment]
Sure thing! However, the .svg file seems that it cannot be displayed without certain plugins. Maybe an edit in the .yml file may help?

**Comment by SuibianP** [comment]
Please try using the [Markdown syntax for images](https://daringfireball.net/projects/markdown/syntax#img), which will handle the relative links correctly. You may also want to refer to [this](https://github.com/ookamitai/utau-zh-docs/blob/45bbd616610c9730483383289f7599097ea940c8/docs/docs/%E5%8E%9F%E9%9F%B3%E8%AE%BE%E5%AE%9A1_oto%E6%9D%A1%E7%9B%AE%E5%88%B0%E5%BA%95%E5%8C%85%E5%90%AB%E4%BA%86%E4%BB%80%E4%B9%88/%E5%8E%9F%E9%9F%B3%E8%AE%BE%E5%AE%9A1.md?plain=1#L11).

As can be seen [here](https://github.com/ookamitai/utau-zh-docs/tree/20bf90ce84cc06eca611c52267f12d01c5014a8a/docs/%E5%8E%9F%E9%9F%B3%E8%AE%BE%E5%AE%9A1_oto%E6%9D%A1%E7%9B%AE%E5%88%B0%E5%BA%95%E5%8C%85%E5%90%AB%E4%BA%86%E4%BB%80%E4%B9%88), the Markdown-compiled HTML document actually lives one level down the hierarchy relative to the image.

**Comment by ookamitai** [comment]
Thanks a lot for your clear explanation! The line has now been replaced with Markdown syntax
:3

**Review by SuibianP — CHANGES_REQUESTED** [review]
Some lists are currently incorrectly rendered due to missing blank line before it. You may find a (rather lengthy) discussion of this issue [here](https://talk.commonmark.org/t/blank-lines-before-lists-revisited/1990).

Also, thanks for the heads up leading to fix f9a59d19e4242a859592c5c06ffedca5cff06a72!

**Review by SuibianP — CHANGES_REQUESTED** [review]
There are still several unrectified lists, including line 57, 70 and 85.

**Comment by ookamitai** [comment]
As I have noticed, some characters are incorrectly highlighted. Any ideas on that?

**Comment by SuibianP** [comment]
This seems to be due to [`InlineHilite`](https://facelessuser.github.io/pymdown-extensions/extensions/inlinehilite/#options) inheriting the [`guess_lang` option from `Highlight`](https://facelessuser.github.io/pymdown-extensions/extensions/highlight/#options).

Meanwhile, there does not seem to be a viable approach to avoid this behaviour while retaining the ability to highlight individual inline snippets if desired, which I will try to propose upstream. For now, this is fine and not a blocker for this PR.

File: mkdocs.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 site_name: utau-zh-docs
 site_description: UTAU相关各种文档的简体中文翻译堆积处。
 site_author: 隋卞
-
+
```
Comment (SuibianP) [inline mkdocs.yml]: Please also remove this trailing whitespace.

**Review by SuibianP — APPROVED** [review]
LGTM. Thank you very much for your contribution!

**Comment by ookamitai** [comment]
My pleasure!

**Pull request merged**